src/XrdPosix/XrdPosixXrootd.hh

```diff
@@ -46,6 +46,7 @@
 
 #include "XrdPosix/XrdPosixOsDep.hh"
 #include "XrdSys/XrdSysPthread.hh"
+#include "XrdPosix/XrdPosixAdmin.hh"
```

Comment:
I'd prefer you forward declare this object. No need to mess up the where PosixXrootd.hh is included as only needed in the cc file.

Reply:
done